sysboundary/setbyuser.cpp

```diff
@@ -288,12 +288,11 @@ namespace SBC {
          std::array<bool, 6> isThisCellOnAFace;
          determineFace(isThisCellOnAFace, mpiGrid, id);
 
-         int max = 6;
-         for (uint i=0; i < max; i++) {
+         for (uint i=6; i < max; i++) {
```

Comment:
`         for (uint i=0; i < 6; i++) {`

Reply:
Goddamnit, what was I thinking there? Or why was I not?
Anyway, fixed.